public void testIssue() {
    JsonMappingException jsonMappingException = expectThrows(JsonMappingException.class, () -> {
        ObjectMapper objectMapper = new ObjectMapper();
        objectMapper.readValue(objectMapper.writeValueAsBytes(new ReferenceChainTest.Outer()), ReferenceChainTest.Outer.class);
    });
    JsonMappingException.Reference reference = jsonMappingException.getPath().get(0);
    assertEquals(ReferenceChainTest.Outer.class, reference.getFrom().getDeclaringClass());
    assertEquals("inner", reference.getFieldName());
}
